aiida_orca/calculations/orca_orca.py

```diff
@@ -65,7 +65,7 @@ def define(cls, spec):
 
         # Exit codes
         spec.exit_code(
-            100, 'ERROR_NO_RETRIEVED_FOLDER', message='The retrieved folder data node could not be accessed.'
+            300, 'ERROR_NO_RETRIEVED_FOLDER', message='The retrieved folder data node could not be accessed.'
```

Comment:
You might want to consider even getting rid of this. If the retrieved folder does not exist, i.e. `self.retrieved` in the `Parser` excepts, there is a serious bug in AiiDA. This hasn't happened yet in all those years. Now this is not to say that this won't ever occur, but you catching the exception won't be that useful anyway. In my own plugins, I don't even bother with this one and just use `self.retrieved` without catching any exceptions.

Reply:
Great, I was actually wondering about this as well. I'll remove it.